x-pack/elastic-agent/pkg/artifact/config.go

```diff
@@ -27,9 +28,6 @@ type Config struct {
 	// TargetDirectory: path to the directory containing downloaded packages
 	TargetDirectory string `json:"targetDirectory" config:"target_directory"`
 
-	// Timeout: timeout for downloading package
-	Timeout time.Duration `json:"timeout" config:"timeout"`
```

Comment:
just for awarness: can be breaking, some customers are using increased timeout to deal with slow network connection and upgrades/download

Reply:
The timeout is not removed for real. It will now be provided by the httpcommon settings, and can still be modified by users. We also keep the default of 120s.